tools/loader/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 
-add_executable(loader
-                 loader.cpp)
-target_link_libraries(loader
+add_executable(image-classifier
+  Loader.cpp
+  image-classifier.cpp)
```

Comment:
I think the convention for cpp/h files is ImageClassifier (ExecutionEngine, etc)